### PR TITLE
PRL-6010: Add error scenarios for payment screen

### PR DESCRIPTION
**Pull request by hcarslaw**
### Jira link
https://tools.hmcts.net/jira/browse/PRL-6010

### Change description
Adds errors for issues with payment completion and case submission.

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/steps/edge-case/pay-your-fee/routeGuard.ts [hunk 1]
```diff
@@ -7,17 +7,23 @@ import { getApplicationFee } from '../../../steps/edge-case/fees/fees-lookup-api
 export const routeGuard = {
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
   get: async (req: AppRequest, res: Response, next: NextFunction) => {
+    if (req.session.paymentError?.hasError === true) {
```
Comment (vivek-sekhar) [inline src/main/steps/edge-case/pay-your-fee/routeGuard.ts:10]: This is not required.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/edge-case/pay-your-fee/routeGuard.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/edge-case/pay-your-fee/routeGuard.ts:10]: If this is removed the help with fees errors get deleted every time the page is loaded, so they never show.

**Review by vivek-sekhar — APPROVED** [review]
